source/Manila/manilaKubernetes.rst

```diff
@@ -0,0 +1,405 @@
+
+    This feature for using Shares with Kubernetes is also currently being tested by the Cloud Team.
+
+
+To use shares on Kubernetes clusters we need the following ``csi`` installed and deployed onto the cluster:
+    - ``manila-csi``: https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/manila-csi-plugin/using-manila-csi-plugin.md
+
+    - ``ceph-csi``: https://github.com/ceph/ceph-csi
+
+The ``manila-csi`` is designed to be lightweight and only handles the interaction between Kubernetes and OpenStack to create, delete, and access shares. 
+The ``ceph-csi`` handles the CephFS side of the filesystem on the Kubernetes cluster. See https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/manila-csi-plugin/developers-csi-manila.md#notes-on-design  for more information on how manila-csi is designed.
+
+Deploying onto the Cluster 
+-----------------------------
+Both the ``manila-csi`` and ``ceph-csi`` can be deployed to a Kubernetes cluster using a Helm chart. 
+
+ceph-csi
+~~~~~~~~~~~
+
+The ``ceph-csi`` can be deployed in the following way:
+
+.. code-block:: bash 
+
+    # Add the chart repository 
+    helm repo add ceph-csi https://ceph.github.io/csi-charts
+    # Create namespace for the chart
+    kubectl create namespace ceph-csi-cephfs
+    # Install the chart
+    helm install --namespace "ceph-csi-cephfs" "ceph-csi-cephfs" ceph-csi/ceph-csi-cephfs
+
+
+You should then be able to see the following pods in the `ceph-csi-cephfs` namespace:
+
+.. code-block:: bash 
+
+    $ kubectl get pods -n ceph-csi-cephfs
+    NAME                                           READY   STATUS    RESTARTS   AGE
+    ceph-csi-cephfs-nodeplugin-6554g               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-6xcfc               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-ddt84               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-h97j5               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-hxj95               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-j65fp               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-ltt7q               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-mhwr8               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-wrjsk               3/3     Running   0          17d
+    ceph-csi-cephfs-nodeplugin-zfl8w               3/3     Running   0          17d
+    ceph-csi-cephfs-provisioner-765854df4c-csdft   5/5     Running   0          17d
+    ceph-csi-cephfs-provisioner-765854df4c-nzp77   5/5     Running   0          17d
+    ceph-csi-cephfs-provisioner-765854df4c-qznz4   5/5     Running   0          17d
+
+
+
+manila-csi
+~~~~~~~~~~~~~~~~~~
+
+Before deploying the Helm chart for ``manila-csi``, we need to modify the ``values.yaml`` file in order to deploy 
```

Comment:
Need to modify -> need to create 

Since the user won't have a copy of this file (unless we've missed a step?)